docs/debug.md

```diff
@@ -1,3 +1,60 @@
+## Grafana Issues
+
+### Grafana is loading but dashbords show no data
+
+Assuming that thanos is working correctly (it is able to ingest and query data). You can proceed with the following steps to debug the issue:
+
+* Check if the Grafana exlorer is working by querying a random metric like `up`.
+  * If it is not working, check communication between the components on the read path using the user token as Grafana does. 
```

Comment:
Maybe add a small reference (or even a link) to the section about the read path communications check?